packages/manager/apps/dedicated/client/app/app.routes.js

```diff
@@ -33,6 +33,28 @@ angular.module('App').config(
       },
     });
 
+    $stateProvider.state('app.userContracts', {
+      url: '/user-contracts',
+      redirectTo: ($transition$) => {
+        const { contracts } = $transition$.params();
+        return contracts ? false : 'app.configuration';
+      },
+      views: {
+        'app@': {
+          component: 'userContracts',
+        },
+      },
+      params: {
+        contracts: null,
+      },
+      translations: { value: ['.'], format: 'json' },
```

Comment:
Should we import translations from `packages/manager/apps/dedicated/client/app/translations` for this state ?